src/main/java/org/gridsuite/modification/server/dto/EquipmentDeletionInfos.java

```diff
@@ -33,6 +36,17 @@ public class EquipmentDeletionInfos extends EquipmentModificationInfos {
     @Schema(description = "Equipment type")
     private String equipmentType;
 
+    @Schema(description = "HVDC having a LCC or not")
+    private boolean hvdcWithLCC;
+
+    @Schema(description = "LCC HVDC converter station Shunt Compensator side 1")
```

Comment:
I think we have a pb of design
This class is a generic class and we add some specific informations for hvdcline

Reply:
yes good point